src/vrct/tests/fs_vrct_test.go

```diff
@@ -13,39 +13,46 @@ func TestCreatingFile(t *testing.T) {
 	}
 	fsVrct := &ruleVrct.Fs
 
-	err = fsVrct.CreateFile("/tmp/spito-test-2137/file.txt", []byte("test value"), false)
+	tmpPath, err := os.MkdirTemp("/tmp", "spito-test-")
```

Comment:
I think it should be "spito-test" and not "spito-test-"
```suggestion
	tmpPath, err := os.MkdirTemp("/tmp", "spito-test")
```

Reply:
MkdirTemp generates random name directory, I wrote `spito-test-` in order to have directory like this: 
`/tmp/spito-test-qwylnj`